api/v1/operator/openapi.yaml

```diff
@@ -0,0 +1,54 @@
+---
+swagger: '2.0'
+info:
+  title: Cilium Operator
+  description: Cilium
+  version: v1beta
+x-schemes:
+- unix
+basePath: "/v1beta"
```

Comment:
Since its an initial version for the operator API I wanted to keep it as `v1beta` similar to what we have for `cilium-health`. I think we can change the base path to `/v1` if everyone agrees.

Reply:
Discussed offline with @aanm 
It makes sense to keep the basepath as `/v1` but the version should be `v1beta`.